node_modules/oae-authentication/lib/api.js

```diff
@@ -16,23 +16,25 @@
 var _ = require('underscore');
 var events = require('events');
 var passport = require('passport');
-
+var crypto = require('crypto');
 var AuthzConstants = require('oae-authz/lib/constants').AuthzConstants;
 var Cassandra = require('oae-util/lib/cassandra');
 var ConfigAPI = require('oae-config');
 var Context = require('oae-context').Context;
 var Locking = require('oae-util/lib/locking');
+var EmailAPI = require('oae-email');
```

Comment:
The dependencies should be in alphabetised groups - first external dependencies, then other OAE module dependencies, then dependencies to the same module.